website/package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "examples": "docusaurus-examples",
     "start": "docusaurus-start",
-    "build:master": "DOCSEARCH_INDEXNAME='meshcloud' DOCSEARCH_APIKEY='1b6387d9fcda3cba4cdb3ffb68cde260' docusaurus-build",
+    "build:master": "DOCSEARCH_INDEXNAME='meshcloud' DOCSEARCH_APIKEY='aa3b874dff5c832fe2e3ed42a8062160' DOCSEARCH_APP_ID='LDDGX81P02' docusaurus-build",
```

Comment:
d: I hope this APIKEY is not sensitive so its okay to have it in the public repo? Not that someone can extract this key and post a own, modified search result to the service? Not sure how it works. I thought during build time it generates an index and transmits to their server? Then to have those keys here I guess could be problematic?

Reply:
![image](https://github.com/meshcloud/meshcloud-docs/assets/14248574/84dc9ae8-6345-4bd0-8ccd-26a9f7c19698)

The key is just for querying in frontend and is safe.